### DISCUSSION TITLE
How to create "switchable" content in RST?

### DISCUSSION THREAD

**Discussion by swaldhoer** [pradyunsg/furo#192] [Questions] [answered]
In the documentation there is often *switchable* content, e.g., see https://pradyunsg.me/furo/reference/api/ 

![image](https://user-images.githubusercontent.com/34184299/126063013-e75b0795-cc21-4a86-9efe-def3e7df8e65.png)

How is this achivied in pure rst, not [md as in in the current docs](https://raw.githubusercontent.com/pradyunsg/furo/34e07e043dcd61553e270c28ad353671219ec33a/docs/reference/api.md)?

**Comment by dgw** [comment, marked as answer]
These are tabs. https://sphinx-inline-tabs.readthedocs.io/en/latest/usage.html

**Reply by swaldhoer** [reply]
Many thanks.

**Reply by pradyunsg** [reply]
They're also documented, although IDK if they were when you started this discussion: https://pradyunsg.me/furo/reference/tabs/
